zeebe/atomix/cluster/src/main/java/io/atomix/cluster/messaging/impl/NettyMessagingService.java

```diff
@@ -468,10 +482,11 @@ public CompletableFuture<Void> stop() {
               openFutures.clear();
             } finally {
               log.info(
-                  "Stopped messaging service bound to {}, advertising {}, and using {}",
+                  "Stopped messaging service bound to {}, advertising {}, and using {} - {}",
                   bindingAddresses,
                   advertisedAddress,
-                  config.isTlsEnabled() ? "TLS" : "plaintext");
+                  config.isTlsEnabled() ? "TLS" : "plaintext",
+                  MDC.get("actor-scheduler"));
```

Comment:
:x: No need to get the actor-scheduler name here. It is already part of the logging context.

Reply:
Reverted.